tests/test_utils_elevation.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import genet.utils.elevation as elevation
+import xarray as xr
+import os
+
+
+elevation_test_folder = os.path.abspath(os.path.join(os.path.dirname(__file__), "test_data", "elevation"))
+tif_path = os.path.join(elevation_test_folder, 'hk_elevation_example.tif')
+array_path = os.path.join(elevation_test_folder, 'elevation_image.nc')
+
+
+def test_output_type_get_elevation_image():
+    img = elevation.get_elevation_image(tif_path)
+    assert type(img).__name__ == 'DataArray'
+
+
+def test_get_elevation_image():
+    output = elevation.get_elevation_image(tif_path)
+    image = xr.open_dataset(array_path)
+    assert output == image
+
+
+def test_get_elevation_data():
+    # need to figure out how to load save and load in an xarray.DataArray - only managed xarray.Dataset
```

Comment:
not sure about this comment being here, but I can try to answer it at least: as I understand it xr.Dataset is a thing that wraps around multiple xr.DataArrays so to get the right data array you need to ask for it from the dataset using the correct index [link](https://docs.xarray.dev/en/stable/generated/xarray.Dataset.html)